source/SIL.AppBuilder.Portal/src/routes/(authenticated)/tasks/[product_id]/+page.server.ts

```diff
@@ -0,0 +1,104 @@
+import type { PageServerLoad } from './$types';
+import { prisma } from 'sil.appbuilder.portal.common';
+
+type Fields = {
+  user?: string;            //Product.Project.Owner.Name
+  email?: string;           //Product.Project.Owner.Email
+  name: string;             //Product.Project.Name
+  description: string;      //Product.Project.Description
+  store?: string;           //Product.Store.Description
+  listingLanguage?: string; //Product.StoreLanguage.Name
+  projectURL?: string;      //Product.Project.WorkflowAppProjectURL
+  product?: string;         //Product.ProductDefinition.Description
+  appType?: string;         //Product.ProductDefinition.ApplicationTypes.Description
+  langCode?: string;        //Product.Project.Language
+}
+
+//replace with Product.Project.Reviewers
+type User = {
+  Id: number;
+  Name: string;
+  Email: string;
+};
+
+export const load = (async ({ params, url, locals }) => {
+  const product = await prisma.products.findUnique({
+    where: {
+      Id: params.product_id
+    },
+    select: {
+      Project: {
+        select: {
+          Name: true,
+          Description: true,
+          WorkflowAppProjectUrl: true,
+          Language: true,
+          Owner: {
+            select: {
+              Name: true,
+              Email: true
+            }
+          },
+          Reviewers: {
+            select: {
+              Id: true,
+              Name: true,
+              Email: true
+            }
+          }
+        }
+      },
+      Store: {
+        select: {
+          Description: true
+        }
+      },
+      StoreLanguage: {
+        select: {
+          Name: true
+        }
+      },
+      ProductDefinition: {
+        select: {
+          Name: true,
+          ApplicationTypes: {
+            select: {
+              Description: true
+            }
+          }
+        }
+      },
+      ProductArtifacts: {
```

Comment:
Currently there is no way to tell if ProductArtifacts is needed or not, because that depends on the state of the workflow, which we haven't implemented yet. Once we've got that implemented I am absolutely in favor of minimalist queries.

Reply:
OK.  Add a comment in the code that states that we would like to consider returning ProductArtifacts only when needed. Honestly, it might be an "early optimization" that isn't needed.  So something like:

Actually, ProductArtifacts should be filters based on the `Products.WorkflowBuildId` = `ProductArtifacts.ProductBuildId`

Reply:
Added some comments for future reference